modules/apps/asset/asset-display-page-api/src/main/java/com/liferay/asset/display/page/portlet/BaseAssetDisplayPageFriendlyURLResolver.java

```diff
@@ -144,30 +143,32 @@ public String getActualURL(
 			infoItemFieldValuesProvider.getInfoItemFieldValues(
 				layoutDisplayPageObjectProvider.getDisplayObject());
 
-		Optional<String> descriptionOptional = _getMappedValueOptional(
+		String description = _getMappedValueOptional(
```

Comment:
@slnn Please remember to rename the method.

Reply:
Sorry, my bad. Forced push with updating.